src/main/java/com/checkout/CsvResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.checkout;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class CsvResponse extends HttpMetadata {
```

Comment:
What you think about naming this class `ContentResponse`? So we can use it for different purposes in the future.

Reply:
Please make it `final` too.